src/metadata_tool/config.clj

```diff
@@ -1,42 +1,42 @@
-;
-; Copyright 2017 Fintech Open Source Foundation
-; SPDX-License-Identifier: Apache-2.0
-;
-; Licensed under the Apache License, Version 2.0 (the "License");
-; you may not use this file except in compliance with the License.
-; You may obtain a copy of the License at
-;
-;     http://www.apache.org/licenses/LICENSE-2.0
-;
-; Unless required by applicable law or agreed to in writing, software
-; distributed under the License is distributed on an "AS IS" BASIS,
-; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-; See the License for the specific language governing permissions and
-; limitations under the License.
-;
+;;
+;; Copyright 2017 Fintech Open Source Foundation
+;; SPDX-License-Identifier: Apache-2.0
+;;
+;; Licensed under the Apache License, Version 2.0 (the "License");;
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+;;
+;;     http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
 
 (ns metadata-tool.config
-  (:require [clojure.string  :as s]
-            [clojure.java.io :as io]
-            [mount.core      :as mnt :refer [defstate]]))
+  (:require
+    [clojure.string  :as s]
+    [clojure.java.io :as io]
+    [mount.core      :as mnt :refer [defstate]]))
 
-; Because java.util.logging is a hot mess
+;; Because java.util.logging is a hot mess
 (org.slf4j.bridge.SLF4JBridgeHandler/removeHandlersForRootLogger)
 (org.slf4j.bridge.SLF4JBridgeHandler/install)
 
 (defstate config
           :start (mnt/args))
```

Comment:
Missing whitespace shenanigans here.

Reply:
comments that need to be aligned with code start with two semicolons, see [here](http://www.lispworks.com/documentation/HyperSpec/Body/02_ddb.htm). That's the same reason behind all the changes in the licensing headers of all files.

Reply:
My comment refers to lines 28 and 29, which don't contain comments:

```clojure
(defstate config
          :start (mnt/args))
```

Elsewhere the PR has removed the horizontal indent prior to `:start` in similar forms.

Reply:
indeed, that was not intentional. I would rather have `:start` be at its normal indent, like in the snippets found in the `mount` documentation itself